src/main/steps/applicant2/you-have-not-confirmed-joint-application/content.ts

```diff
@@ -0,0 +1,20 @@
+import { TranslationFn } from '../../../app/controller/GetController';
+
+const en = ({ isDivorce, partner, civilPartner }) => ({
+  title: 'You have not confirmed your joint application',
+  line1: `You have stated that your ${
+    isDivorce ? 'marriage' : 'civil partnership'
+  } has not irretrievably broken down. Therefore you cannot make a joint application for divorce.`,
+  line2: `Your ${isDivorce ? partner : civilPartner} has been notified by email.`,
```

Comment:
```suggestion
  line2: `Your ${partner} has been notified by email.`,
```
i believe there is a function that set partner to husband/wife/civil partner so there isnt a need for the check